### ISSUE TITLE
Console progress bar takes two lines instead of one lines

### ISSUE THREAD

**Issue by ivicts** [fastai/fastprogress#30]
![image](https://user-images.githubusercontent.com/19903139/54441635-21c95100-4778-11e9-83ff-4a2e34dba21c.png)

Is this a bug or a feature?
I am not sure why I got two lines of the progress bar in my codes. I have tried running the Examples.ipynb and it only has one line for the console progress bar.

How to make it display only on one line instead of two?

**Comment by sgugger** [comment]
That's definitely not the expected behavior. Can you share a little bit more about your enviromnent and your code?

**Comment by ivicts** [comment]
![image](https://user-images.githubusercontent.com/19903139/54506817-4e988680-4979-11e9-9f95-b3e5e0e09d2b.png)

Codes:

`from fastprogress import force_console_behavior
master_bar, progress_bar = force_console_behavior()

def train(net, optimizer, opt_parameters, master_bar):
    """
    train one epoch
    """

    net.train()
    
    
    running_loss = 0.0
    running_mae = 0.0
    running_nb_data = 1
    running_nb_batch = 0
    total_batch = 128


    for batch_num in progress_bar(range(total_batch), parent = master_bar):

        sleep(1)
        current_batch_size = 100

        # Log intermediate statistics
        result = 'Train loss:{loss:.4f} mae:{mae:.4f}'.format(loss = running_loss/running_nb_data,
          mae = running_mae/running_nb_data)
        master_bar.child.comment = result


    # loss and unnorm_mae values for one epoch
    loss = running_loss/ running_nb_data
    mae = running_mae/ running_nb_data
    
    return loss, mae
###############  
    

if True:
    epoch_bar = master_bar(range(2))
    for epoch in epoch_bar:
        train_loss, train_mae = train(net, optimizer, opt_parameters, epoch_bar)
        print(train_loss, train_mae)
`

Environment: (I have tried running the Examples.ipynb and it only has one line for the console progress bar using this same environment.)

`# This file may be used to create an environment using:
# $ conda create --name <env> --file <this file>
# platform: linux-64
_tflow_1100_select=0.0.1=gpu
absl-py=0.4.1=py36_0
appdirs=1.4.3=py36h28b3542_0
ase=3.17.0=py_0
asn1crypto=0.24.0=py36_0
astor=0.7.1=py36_0
attrs=18.2.0=py36h28b3542_0
automat=0.7.0=py36_0
backcall=0.1.0=py36_0
blas=1.0=mkl
bleach=2.1.4=py36_0
bzip2=1.0.6=h14c3975_5
ca-certificates=2019.1.23=0
cairo=1.14.12=h8948797_3
certifi=2019.3.9=py36_0
cffi=1.11.5=py36he75722e_1
chardet=3.0.4=py36_1
click=7.0=py_0
constantly=15.1.0=py36h28b3542_0
cryptography=2.6.1=py36h1ba5d50_0
cudatoolkit=9.0=h13b8566_0
cudnn=7.1.2=cuda9.0_0
cupti=9.0.176=0
cycler=0.10.0=py36_0
dbus=1.13.2=h714fa37_1
decorator=4.3.0=py36_0
entrypoints=0.2.3=py36_2
expat=2.2.6=he6710b0_0
fastcache=1.0.2=py36h14c3975_2
fastprogress=0.1.20=py_0
flask=1.0.2=py_2
fontconfig=2.13.0=h9420a91_0
freetype=2.9.1=h8a8886c_1
fribidi=1.0.5=h7b6447c_0
gast=0.2.0=py36_0
git=2.20.1=pl526hacde149_0
glib=2.56.2=hd408876_0
gmp=6.1.2=h6c8ec71_1
gmpy2=2.0.8=py36h10f8cd9_2
graphite2=1.3.12=h23475e2_2
graphviz=2.40.1=h21bd128_2
grpcio=1.12.1=py36hdbcaa40_0
gst-plugins-base=1.14.0=hbbd80ab_1
gstreamer=1.14.0=hb453b48_1
harfbuzz=1.8.8=hffaf4a1_0
html5lib=1.0.1=py36_0
hyperlink=18.0.0=py36_0
icu=58.2=h9c2bf20_1
idna=2.7=py36_0
incremental=17.5.0=py36_0
intel-openmp=2019.0=118
ipykernel=4.10.0=py36_0
ipython=6.5.0=py36_0
ipython_genutils=0.2.0=py36_0
ipywidgets=7.4.2=py36_0
itsdangerous=1.1.0=py_0
jedi=0.12.1=py36_0
jinja2=2.10=py36_0
jpeg=9b=h024ee3a_2
jsonschema=2.6.0=py36_0
jupyter=1.0.0=py36_7
jupyter_client=5.2.3=py36_0
jupyter_console=5.2.0=py36_1
jupyter_core=4.4.0=py36_0
kiwisolver=1.0.1=py36hf484d3e_0
krb5=1.16.1=h173b8e3_7
libboost=1.65.1=habcd387_4
libcurl=7.64.0=h20c2e04_2
libedit=3.1.20181209=hc058e9b_0
libffi=3.2.1=hd88cf55_4
libgcc-ng=8.2.0=hdf63c60_1
libgfortran-ng=7.3.0=hdf63c60_0
libpng=1.6.36=hbc83047_0
libprotobuf=3.6.0=hdbcaa40_0
libsodium=1.0.16=h1bed415_0
libssh2=1.8.0=h1ba5d50_4
libstdcxx-ng=8.2.0=hdf63c60_1
libtiff=4.0.9=he85c1e1_2
libuuid=1.0.3=h1bed415_2
libxcb=1.13=h1bed415_1
libxml2=2.9.8=h26e45fe_1
markdown=2.6.11=py36_0
markupsafe=1.0=py36h14c3975_1
matplotlib=3.0.3=py36h5429711_0
mistune=0.8.3=py36h14c3975_1
mkl=2018.0.3=1
mkl_fft=1.0.6=py36h7dd41cf_0
mkl_random=1.0.1=py36h4414c95_1
monty=1.0.3=py36_2
mpc=1.1.0=h10f8cd9_1
mpfr=4.0.1=hdf1c602_3
mpmath=1.0.0=py36_2
nbconvert=5.3.1=py36_0
nbformat=4.4.0=py36_0
nccl=1.3.5=cuda9.0_0
ncurses=6.1=he6710b0_1
networkx=2.1=py36_0
ninja=1.8.2=py36h6bb024c_1
notebook=5.6.0=py36_0
numpy=1.15.4=py36h1d66e8a_0
numpy-base=1.15.4=py36h81de0dd_0
olefile=0.46=py36_0
openssl=1.1.1b=h7b6447c_1
palettable=3.1.1=py36_2
pandas=0.23.4=py36h04863e7_0
pandoc=2.2.3.2=0
pandocfilters=1.4.2=py36_1
pango=1.42.4=h049681c_0
parso=0.3.1=py36_0
patsy=0.5.0=py36_0
pcre=8.42=h439df22_0
perl=5.26.2=h14c3975_0
pexpect=4.6.0=py36_0
pickleshare=0.7.4=py36_0
pillow=5.2.0=py36heded4f4_0
pip=10.0.1=py36_0
pixman=0.34.0=hceecf20_3
prometheus_client=0.3.1=py36h28b3542_0
prompt_toolkit=1.0.15=py36_0
protobuf=3.6.0=py36hf484d3e_0
ptyprocess=0.6.0=py36_0
py-boost=1.65.1=py36hf484d3e_4
pyasn1=0.4.4=py36h28b3542_0
pyasn1-modules=0.2.2=py36_0
pycparser=2.18=py36_1
pydispatcher=2.0.5=py36h30c4b39_1
pygments=2.2.0=py36_0
pymatgen=2018.9.12=py36_0
pyopenssl=18.0.0=py36_0
pyparsing=2.2.0=py36_1
pyqt=5.9.2=py36h05f1152_2
pysocks=1.6.8=py36_0
python=3.6.8=h0371630_0
python-dateutil=2.7.3=py36_0
pytorch=0.4.1=py36ha74772b_0
pytz=2018.5=py36_0
pyzmq=17.1.2=py36h14c3975_0
qt=5.9.7=h5867ecd_1
qtconsole=4.4.1=py36_0
rdkit=2018.03.4.0=py36h71b666b_1
readline=7.0=ha6073c6_4
requests=2.20.1=py36_0
ruamel.yaml=0.15.50=py36_0
scikit-learn=0.20.0=py36h4989274_1
scipy=1.1.0=py36hfa4b5c9_1
seaborn=0.9.0=py36_0
send2trash=1.5.0=py36_0
service_identity=17.0.0=py36h28b3542_0
setuptools=40.2.0=py36_0
simplegeneric=0.8.1=py36_2
sip=4.19.8=py36hfc679d8_0
six=1.11.0=py36_1
spglib=1.10.4.7=py36h39e3cac_0
sqlite=3.27.2=h7b6447c_0
statsmodels=0.9.0=py36h035aef0_0
sympy=1.3=py36_0
tabulate=0.8.2=py36_2
tensorboard=1.10.0=py36hf484d3e_0
tensorflow=1.10.0=gpu_py36h97a2126_0
tensorflow-base=1.10.0=gpu_py36h6ecc378_0
tensorflow-gpu=1.10.0=hf154084_0
termcolor=1.1.0=py36_1
terminado=0.8.1=py36_1
testpath=0.3.1=py36_0
tk=8.6.8=hbc83047_0
torchvision=0.2.1=py36_1
tornado=5.1=py36h14c3975_0
tqdm=4.31.1=py_0
traitlets=4.3.2=py36_0
twisted=18.7.0=py36h14c3975_1
urllib3=1.23=py36_0
wcwidth=0.1.7=py36_0
webencodings=0.5.1=py36_1
werkzeug=0.14.1=py36_0
wheel=0.31.1=py36_0
widgetsnbextension=3.4.1=py36_0
xz=5.2.4=h14c3975_4
zeromq=4.2.5=hf484d3e_1
zlib=1.2.11=ha838bed_2
zope=1.0=py36_1
zope.interface=4.5.0=py36h14c3975_0`

**Comment by sgugger** [comment]
Should be fixed in master now.
